dist/index.html

```diff
@@ -2,10 +2,55 @@
 <html>
   <head>
     <meta charset="UTF-8">
+    <link rel="stylesheet" href="../src/styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Mogra&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Kalam:wght@300;400;700&display=swap" rel="stylesheet">    
     <title>Whats Cookin</title>
   </head>
   <body>
-    <h1>Whats Cookin</h1>
+    <header>
+      <nav class="nav-bar">
+        <section id="user-search">
+            <button type="button" id="search-btn"></button>
+            <figure>
+              <img id="user" src="../src/images/user.png">
+              <figcaption id="user-initials">AB</figcaption>
+            </figure>
+        </section>
+        <section class="nav-pages">
+          <button type="button" id="home-icon"></button>
+          <button type="button" class="nav-btn" id="saved-btn">Saved Recipes</button>
+          <button type="button" class="nav-btn" id="all-recipes-btn">All Recipes</button>
+          <button type="button" class="nav-btn" id="all-categories-btn">All Categories</button>
+        </section>
+      </nav>
+    </header>
+    <main>
+      <section id="home-view">
+        <figure id="home-banner">
+          <img id="recipe-of-the-day" src="https://spoonacular.com/recipeImages/595736-556x370.jpg">
```

Comment:
We should have an alt attribute to ensure our logo can be described by screen readers.

Reply:
Absolutely! I will add alt tags to both!